packages/react-data-grid/src/Cell/CellAction.tsx

```diff
@@ -10,12 +10,17 @@ export interface CellActionProps extends CellActionButton {
 export default function CellAction({ icon, actions, callback, isFirst }: CellActionProps) {
   const [isOpen, setIsOpen] = useState(false);
 
-  const cellActionClasses = classNames('rdg-cell-action', {
-    'rdg-cell-action-last': isFirst
-  });
-  const actionButtonClasses = classNames('rdg-cell-action-button', {
-    'rdg-cell-action-button-toggled': isOpen
-  });
+  const cellActionClasses = useMemo(() => {
+    return classNames('rdg-cell-action', {
+      'rdg-cell-action-last': isFirst
+    });
+  }, [isFirst]);
+
+  const actionButtonClasses = useMemo(() => {
+    return classNames('rdg-cell-action-button', {
+      'rdg-cell-action-button-toggled': isOpen
+    });
+  }, [isOpen]);
```

Comment:
If you _really_ want to optimize these you could just do
```ts
const actionButtonClasses = isOpen
  ? 'rdg-cell-action-button rdg-cell-action-button-toggled'
  : 'rdg-cell-action-button';
```
It might be worth it for simple cases like this one. 🤔 wdyt?

Reply:
I am okay with your suggestion but for now I ended up removing this optimization. I will add it back if needed after we profile the grid. I do remember `classNames` was taking some time and this optimization will be useful for Cell components